### ISSUE TITLE
Open Source Mycetoma (MycetOS) Meeting May XX 2021

### ISSUE THREAD

**Issue by fantasy121** [OpenSourceMycetoma/Series-1-Fenarimols#54]
Hi All,
Our next meeting is on (Date and Time TBC @kym834 ). Kymberley will update this when she's back from leave.

We'll be meeting at Zoom ID: 992-8154-9497 https://uni-sydney.zoom.us/j/99281549497.

Times: TBA

Recording of this meeting to go here

Previous meeting are here (#50 )

Agenda

- [ ] ISNTD talk event @mattodd @wwjvdsande. Also see #53 
- [ ] Check Master List for Duplicate compounds with different MYOS codes and merge them accordingly @dmitrij176
- [ ] Confirm isomeric forms of MYOS_00004_00_01, MYOS_00010_00_01, MYOS_00117_00_01 and MYOS_00138_00_01
- [ ] Find a way to specify and retain isomer information in Master List
- [ ] Kymberley on update of Series 2 compounds @kym834 
- [ ] @fantasy121 Hung SAR/synthetic update. Upload Chemdraw files for SAR, share with group
- [ ] @fantasy121 Hung will ship synthetic batch to Wendy. Add to MYOS list.
- [ ] @fantasy121 New roster for host/minute taking by institution. See #52 
- [ ] Erasmus University, update next lots of biological evaluation @wwjvdsande @MA-Jjingyi @Wilson-Lm 
- [ ] UCL synthetic update @dmitrij176 
- [ ] Kaitlin will look into developing some materials for her students to start exploring the SAR of some of the Fenarimol analogues. @kbea282

Any other business/Closing remarks

@OpenSourceMycetoma/corecontrib, please add anything you wish to discuss to the agenda

**Comment by fantasy121** [comment]
I've worked out the isomeric forms of the following ( I've added a note on the master list of their isomeric identity):

MYOS_00004_00_01 _(S)_ enantiomer version of EPL-BS1246
MYOS_00010_00_01 racemate version of EPL-BS1025
MYOS_00117_00_01 _(S)_ enantiomer version of EPL-BS1246
MYOS_00138_00_01 DM16-1 ?

Can you confirm that 00138 is a racemate? @dmitrij176 

We need a better solution to retain isomer information in our Master List as this mistake keeps happening. You can also see that the isomers have different efficacy as well so best not to mix them up in the long run. Any thoughts on how to do this efficiently? @mattodd @dmitrij176

@dmitrij176 Can I also put you in charge of finding instances where the same chemical structure is assigned multiple unique MYOS_00000.... codes? If you can group each of these duplicates under the one 00000 code and assign batches or some other way to denote that they're the same chemical structure. My understanding is that new 00000 numberings are meant to be assigned to unique chemical structures only. You can download my chemdraw zip file at #51 to view all the structures so far (otherwise let me know so I can give you PNG versions of these files to view)

**Comment by mattodd** [comment]
Yes @fantasy121 this looks important to sort out - we can't have duplicative numbers. The codes we're using do not have something for enantiomeric excess, so it's easiest to give racemate and enantiomer separate codes. So those ones you highlighted above need correcting, and any other instances of this need to be identified.

**Comment by dmitrij176** [comment]
@fantasy121 @mattodd  MYOS_00138 is definitely a racemate. So I though that we all agreed that isomers/enantiomers would be noted in the **aka** section, because we havent devised any MYOS convention in the context of the MYOS code.

Regarding duplicates, yes @fantasy121 no problem. I will check the list.

**Comment by mattodd** [comment]
"Different stereoisomers (or mixtures thereof) should have different MYOS codes (i.e. a difference in the first 5 numbers)" was how we have it in [the numbering convention](https://github.com/OpenSourceMycetoma/MycetOS-Tech-Ops/wiki/The-Molecule-Numbering-Convention).
Obviously if we have to delete a number because it's a duplicate then we just delete it and we don't renumber all the later compounds. It doesn't matter if a MYOS code that was deleted is not then taken up by another compound.

**Comment by mattodd** [comment]
Hi all @OpenSourceMycetoma/corecontrib - any update on the date/time of the meeting (https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/50#issuecomment-839665973)? Currently assuming 25th at 10 pm London?

**Comment by Wilson-Lm** [comment]
If I recall what was agreed in the last meeting, we will now stick to the same time for the next few meetings or until changed. The meeting will be on 25th on this time:

Times: Sydney - 9 PM, Amsterdam - 1 PM, London - 12 Noon, USA (New York) - 7 AM.

**Comment by fantasy121** [comment]
Date and time for our May meeting is now confirmed, please look at OG post @OpenSourceMycetoma/corecontrib @bendndi @alintheopen @pedrochemicals @kbea282

**Comment by MFernflower** [comment]
@kym834 Perhaps one of the students you are working with could prepare the oxime I proposed?

**Comment by MFernflower** [comment]
@kym834 The student could perhaps try making the oxime by grinding: https://orgmedchemlett.springeropen.com/articles/10.1186/2191-2858-1-12

**Comment by fantasy121** [comment]
@OpenSourceMycetoma/corecontrib sorry I’m an apology for tonight. I was filming for a science communication project but went overtime. I won’t make it in time for our meeting. I will work on finishing the rest of my SAR this week and post on GHI

**Comment by MFernflower** [comment]
@dmitrij176 
![ketone](https://user-images.githubusercontent.com/3164942/119489360-9c636300-bd29-11eb-9df7-e4ec193d79e0.png)

**Comment by MFernflower** [comment]
From today's meeting - a possible trio of oximes to follow after pyrifenox: @dmitrij176 

![oximesMay25](https://user-images.githubusercontent.com/3164942/119546882-67273700-bd62-11eb-8148-b79bf51cc67c.png)

**Comment by MFernflower** [comment]
https://wi.knaw.nl/page/fungal_display/8589

@wwjvdsande

Boydii is commercially available from an NL supplier


I think the strain used is a white grain mycetoma isolate.

**Comment by MFernflower** [comment]
@dmitrij176 

finished oxime proposal list:

![119546882-67273700-bd62-11eb-8148-b79bf51cc67c](https://user-images.githubusercontent.com/3164942/119760237-a2675a00-be77-11eb-85de-c431054fcef6.png)

**Comment by dmitrij176** [comment]
@kym834 could you please add some additional information that you think might be relevent to the minutes section, under the university of Sydney paragraph. You will see several brackets as you go through it. Just 1-2 sentences with details on group variation and the hit compound.
Also, could you send me the info/links on the synthesis that we talked about last time.
Thanks in advance.

**Comment by MFernflower** [comment]
@dmitrij176 what one of the three proposed oximes are you going to make?

**Comment by kym834** [comment]
> @kym834 could you please add some additional information that you think might be relevant to the minutes section, under the university of Sydney paragraph. You will see several brackets as you go through it. Just 1-2 sentences with details on group variation and the hit compound.
> Also, could you send me the info/links on the synthesis that we talked about last time.
> Thanks in advance.

**Synthesis**
Step 1 -  lithium halogen exchange and substitution (KRS_015)
https://github.com/TheBreakingGoodProject/ELN-Kymberley-Scroggie/issues/24
Step 2 - TEMPO oxidation (KRS_003)
https://github.com/TheBreakingGoodProject/ELN-Kymberley-Scroggie/issues/11. Note that this oxidation is dependent on pH and temperature. If you keep the ratio of buffer to solvent to bleach you shouldn't have any problems. TEMPO radical decomposes rapidly at 25 °C so keep below 10 (lower is fine but reaction slower).

I will post an update with all the chemical structures and details into the Series 2 repo and link the minutes above to this and update the brackets. 

Thanks for your write up of the minutes @dmitrij176!

**Comment by dmitrij176** [comment]
> > @kym834 could you please add some additional information that you think might be relevant to the minutes section, under the university of Sydney paragraph. You will see several brackets as you go through it. Just 1-2 sentences with details on group variation and the hit compound.
> > Also, could you send me the info/links on the synthesis that we talked about last time.
> > Thanks in advance.
> 
> **Synthesis**
> Step 1 - lithium halogen exchange and substitution (KRS_015)
> [TheBreakingGoodProject/ELN-Kymberley-Scroggie#24](https://github.com/TheBreakingGoodProject/ELN-Kymberley-Scroggie/issues/24)
> Step 2 - TEMPO oxidation (KRS_003)
> [TheBreakingGoodProject/ELN-Kymberley-Scroggie#11](https://github.com/TheBreakingGoodProject/ELN-Kymberley-Scroggie/issues/11). Note that this oxidation is dependent on pH and temperature. If you keep the ratio of buffer to solvent to bleach you shouldn't have any problems. TEMPO decomposes rapidly at 25 °C so keep at 10°C (lower is fine but reaction slower).
> 
> I will post an update with all the chemical structures and details into the Series 2 repo and link the minutes above to this and update the brackets.
> 
> Thanks for your write up of the minutes @dmitrij176!

Thank you Kymberley.
